integration/src/main/resources/configs/integration/ListWorkspaces.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ListWorkspaces",
+  "description": "List workspaces",
+  "serverSpecificationFile": "workspace-dev.json",
+  "kubernetes": {},
+  "application": {},
+  "testScripts": [
+    {
+      "name": "ListWorkspaces",
+      "numberOfUserJourneyThreadsToRun": 1,
+      "userJourneyThreadPoolSize": 1,
+      "expectedTimeForEach": 5,
+      "expectedTimeForEachUnit": "SECONDS",
+      "parameters": ["wm-default-spend-profile"]
+    }
+  ],
+  "testUserFiles": ["bella.json", "liam.json"]
```

Comment:
I believe liam is registered in relevant Sam environments (particularly alpha and staging), but it's worth checking that bella is also registered.

Reply:
Just double-checked, Liam is registered in both alpha and staging but Bella was only registered in alpha.
Per your other comment, I registered Bella in staging so we can run this in any of those envs.